BLSQ/iaso
# IA-2502: Fix not blocking issues related to merged ticket  https://bluesquare.atlassian.net/browse/IA-2345

Explain what problem this PR is resolving
- There are comment made on the [PR](https://github.com/BLSQ/iaso/pull/872)  
- This PR is for to fix them
Related JIRA tickets : [IA-2502](https://bluesquare.atlassian.net/browse/IA-2502)

## Self proofreading checklist


## Changes
- Remove some code duplication
- Avoid permissions duplication while creating a user
- Add an information text on modules page

## How to test
- Go on the modules pages
- Try to set up an account through the api 
- And check if the user related to the account has the right modules

## Print screen / video
![Screenshot from 2023-10-26 17-13-41](https://github.com/BLSQ/iaso/assets/19631540/1fc2bd52-a1ea-4b42-8375-7e9e3a355e66)


## Notes
This purpose of this PR is to refactor some code from comments made in this [PR](https://github.com/BLSQ/iaso/pull/872)

[IA-2502]: https://bluesquare.atlassian.net/browse/IA-2502?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
